modules/compliance-supported-profiles.adoc

```diff
@@ -86,12 +86,14 @@ The Compliance Operator provides the following compliance profiles:
 |0.1.47+
 |link:https://www.pcisecuritystandards.org/document_library?document=pci_dss[PCI Security Standards &#174; Council Document Library]
 |`x86_64`
+ `ppc64le`
 
 |ocp4-pci-dss-node
 |PCI-DSS v3.2.1 Control Baseline for Red Hat OpenShift Container Platform 4
 |0.1.47+
 |link:https://www.pcisecuritystandards.org/document_library?document=pci_dss[PCI Security Standards &#174; Council Document Library]
 |`x86_64`
+ `ppc64le`
```

Comment:
@prb112 As a part of the next downstream Compliance Operator release (0.1.57) I have included Power support to the PCI profiles as previously discussed. You can view a preview [here](http://file.rdu.redhat.com/antaylor/co-0.1.55-rns/security/compliance_operator/compliance-operator-supported-profiles.html#compliance-supported-profiles_compliance-operator-supported-profiles). Please let me know if I need to make any further changes!

Reply:
@rhmdnd for visibility ^

Reply:
awesome - thank you